rules/windows/process_creation/proc_creation_win_filefix_browsers.yml

```
@@ -31,7 +40,7 @@ detection:
             - '\powershell.exe'
             - '\pwsh.exe'
             - '\regsvr32.exe'
-        CommandLine|contains: '#'
+        CommandLine|re: '(^".+".*|#)'
```

```suggestion
        CommandLine|re: '(^".+".*|#)'
```
should we also use the marker 'http' in OR condition @phantinuss?

---

Don't like unbound regex tbh. But I don't see an alternative right now... http could be added, yes. And then don't need the double quotes regex.